target/linux/ramips/dts/mt7620a_edimax_br-6208ac-v2.dts

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * Copyright (C) 2023 Stefan Weil <sw@weilnetz.de>
+ */
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "edimax,br-6208ac-v2", "ralink,mt7620a-soc";
+	model = "Edimax BR-6208AC v2";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_firmware;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		// Vendor firmware: /proc/RESET_BUTTON
+		reset_wps {
+			label = "reset_wps";
+			// TODO: test
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	// Vendor LED configuration:
+	// crw-r--r--    1  166,  39 Mar 27 09:52 Internet_LED
+	// crw-r--r--    1  166,  41 Mar 27 09:52 OpenVPN_LED
+	// crw-r--r--    1  166,  42 Mar 27 09:52 NewFW_LED
+	// crw-r--r--    1  166,  43 Mar 27 09:52 ABAND_LED
+	// crw-r--r--    1  166,  44 Mar 27 09:52 GBAND_LED
+
+	// Vendor firmware: /proc/driver/led
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "green:power";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_POWER;
+			gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+		};
+
+		// RALINK_REG_PIO3924DATA 39-24
+		led_internet: internet {
+			label = "green:internet";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		};
+
+		// RALINK_REG_PIO7140DATA 44-40
+		led_wlan_2_4ghz: wlan_2_4ghz {
+			label = "green:wlan2g";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WLAN;
+			gpios = <&gpio2 4 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1radio";
+		};
+
+		// RALINK_REG_PIO7140DATA 43-40
+		led_wlan_5ghz: wlan_5ghz {
+			label = "green:wlan5g";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_WLAN;
+			gpios = <&gpio2 3 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0radio";
+		};
+
+		// RALINK_REG_PIO7140DATA 42-40
+		led_firmware: firmware {
+			label = "green:firmware";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio2 2 GPIO_ACTIVE_LOW>;
+			// default-state = "keep";
+		};
+
+		// RALINK_REG_PIO7140DATA 41-40
+		led_vpn: vpn {
+			label = "green:vpn";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+		usb-power {
+			gpio-export,name="usb-power";
+			gpio-export,output=<1>;
+			gpios = <&gpio2 5 GPIO_ACTIVE_HIGH>;
+		};
+	};
+};
+
+&gdma {
+	status = "okay";
+};
+
+// Vendor firmware: RALINK_REG_PIO7140 - gpio0: gpio@600
+// Vendor firmware: RALINK_REG_PIO3924 - gpio1: gpio@638
+// Vendor firmware: RALINK_REG_PIO7140 - gpio2: gpio@660
+// Vendor firmware: RALINK_REG_PIO72   - gpio3: gpio@688
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&i2c {
+//	status = "okay";
+};
+
+&pcm {
+	status = "okay";
+};
```

Comment:
No PCM H/W onboard?

Reply:
Probably not.

Reply:
If there is no such HW please do not activate this node.

Reply:
Done.